code_to_gast/loop/py_loop.py

```diff
@@ -1,8 +1,128 @@
 import py_router as pr
+import ast
 
 def while_statement_to_gast(node):
     gast = {}
     gast["type"] = "whileStatement"
     gast["body"] = pr.node_to_gast(node.body)
     gast["test"] = pr.node_to_gast(node.test)
-    return gast
\ No newline at end of file
+    return gast
+
+"""
+Converts python for statements to gast
+Includes:
+    ForRangeStatements: for i in range(0,10,1)
+    ForOfStatements: for elem in [1,2,3]
+"""
+def for_statement_to_gast(node):
+    if type(node.iter) == ast.Call:
+        return for_range_statement_to_gast(node)
+    else: 
+        return for_of_statement_to_gast(node)
+
+def for_of_statement_to_gast(node):
+    gast = {}
+    gast["type"] = "forOfStatement"
+    gast["init"] = pr.node_to_gast(node.target)
+    gast["iter"] = pr.node_to_gast(node.iter)
+    gast["body"] = pr.node_to_gast(node.body)
+    return gast
+
+def for_range_statement_to_gast(node):
+    gast = {}
+    gast["type"] = "forRangeStatement"
+
+    args = node.iter.args
+    if len(args) < 2 or len(args) > 3:
```

Comment:
Can we invert this so this is the final else clause?

Reply:
Yes- I'll also pull it into separate function as per https://github.com/jackdavidweber/cjs_capstone/pull/60#discussion_r451866723